infra/lib/ci-cluster.ts

```diff
@@ -15,26 +12,24 @@ import {
   PROW_JOB_NAMESPACE,
   PROW_NAMESPACE,
 } from "./test-ci-stack";
-import { KubernetesManifest } from "@aws-cdk/aws-eks";
-import { Chart } from "cdk8s";
 
 export type CIClusterCompileTimeProps = ProwGitHubSecretsChartProps;
 
 export type CIClusterRuntimeProps = {};
 
 export type CIClusterProps = CIClusterCompileTimeProps & CIClusterRuntimeProps;
 
-export class CICluster extends cdk.Construct {
+export class CICluster extends Construct {
   readonly testCluster: eks.Cluster;
   readonly testNodegroup: eks.Nodegroup;
 
   readonly namespaceManifests: eks.KubernetesManifest[];
 
-  constructor(scope: cdk.Construct, id: string, props: CIClusterProps) {
+  constructor(scope: Construct, id: string, props: CIClusterProps) {
     super(scope, id);
 
     this.testCluster = new eks.Cluster(scope, "TestInfraCluster", {
-      version: eks.KubernetesVersion.V1_21,
+      version: eks.KubernetesVersion.V1_24,
```

Comment:
Will CDK handle small steps of version-to-version upgrades? or did you make bumps one by one?

Reply:
I actually think a jump like this would not be possible if we were upgrading the cluster, but with our plan to recreate the cluster I don't think it'll matter